cherrypy/test/test_states.py

```diff
@@ -110,7 +112,7 @@ def test_0_NormalStateFlow(self):
 
         host = cherrypy.server.socket_host
         port = cherrypy.server.socket_port
-        self.assertRaises(IOError, cherrypy._cpserver.check_port, host, port)
+        self.assertRaises(IOError, portend._check_port, host, port, timeout=0.1)
```

Comment:
Codacy is right. `portend` should expose public API instead of private functions.